.github/workflows/release.yml

```diff
@@ -49,5 +49,5 @@ jobs:
           CHECKOUT_BRANCH_NAME: ${{ github.ref_name }}
           PR_BRANCH_NAME: ${{ steps.createPrBranchName.outputs.PR_BRANCH_NAME }}
         run: |
-          gh pr create --title "Release $RELEASE_VERSION" --body "" --base "$CHECKOUT_BRANCH_NAME" --reviewer ${{ github.actor }}
+          gh pr create --title "Release $RELEASE_VERSION" --body "" --base "$CHECKOUT_BRANCH_NAME" --reviewer ${{ github.actor }} --head $(git branch $PR_BRANCH_NAME)
```

Comment:
should not this just be
`--head $PR_BRANCH_NAME`
?

Reply:
It would have failed so badly! Fixed it now! Thanks!